scripts/one-off/records-migration/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
it's usually fair to have a `scripts/package.json` so that it can benefits more than one